src/main/java/ch/njol/skript/effects/EffChange.java

```diff
@@ -73,8 +73,11 @@
 		"delete {variable}",
 		"# reset:",
 		"reset walk speed of player",
-		"reset chunk at the targeted block"})
-@Since("1.0 (set, add, remove, delete), 2.0 (remove all)")
+		"reset chunk at the targeted block",
+		"# toggle:",
```

Comment:
Not on the right class, maybe revert changes to this file?

Reply:
Ups, forgot to remove that 😅